pftp/data_handler.go

```diff
@@ -218,6 +224,13 @@ func (d *dataHandler) Close() error {
 		}
 	}
 
+	if d.chanWaiting {
```

Comment:
what does mean `chanWaiting`?
I guess `chan` mean is  `channel`, please consider good name more than now.